src/main/java/creativei/helper/ResponseHelper.java

```diff
@@ -44,7 +40,7 @@ public static InquiryVo getCreateInquiryResponseData(Inquiry inquiry, InquiryVo
         if (inquiryVo.getAddress() != null)
             inquiryVo.setAddress(getCreateAddressResponseData(inquiry.getInquiryAddress(), inquiryVo.getAddress()));
         if (inquiryVo.getEducation() != null)
-            inquiryVo.setEducation(getCreateEducationResponseData(inquiry.getInquiryEducation(), new ArrayList<EducationVo>() {}));
+            inquiryVo.setEducation(getCreateEducationResponseData(inquiry.getInquiryEducations(), new ArrayList<EducationVo>() {}));
```

Comment:
Why are we passing empty array as perameter?

Reply:
if i take inquiryVo.getEducation() it adds this field that are to be sent in response.So if one educationVo is there in request.It provides two same educatioVO in response thats why it is empty.

Reply:
removed as parameter added in  method.